core/templates/pages/collection-player-page/collection-player-page.directive.ts

```diff
@@ -238,7 +239,8 @@ angular.module('oppia').directive('collectionPlayerPage', [
           };
           ctrl.$onInit = function() {
             $scope.$watch('$ctrl.collection', function(newValue) {
-              if (newValue !== null) {
+              // eslint-disable-next-line max-len
```

Comment:
Please avoid pragmas if possible.

In this case, we can use a variable to reduce the line length.

Reply:
Done. I reduced the line length and removed the pragmas.